packages/desktop-client/src/components/modals/ImportTransactionsModal/utils.js

```
@@ -0,0 +1,187 @@
+      parts = twoDig(str);
+      year = `20${parts[0]}`;
+      month = parts[1];
+      day = parts[2];
+      break;
+    case 'mm dd yy':
+      parts = twoDig(str);
+      year = `20${parts[2]}`;
+      month = parts[0];
+      day = parts[1];
+      break;
+    default:
```

Biome

[error] The default clause should be the last switch clause.

The following case clause is here:

Regardless its position, the default clause is always executed when there is no match. To avoid confusion, the default clause should be the last switch clause.

(lint/suspicious/useDefaultSwitchClauseLast)